simplified-bookmarks-api/src/main/java/org/nypl/simplified/bookmarks/api/BookmarkAnnotationsJSON.kt

```diff
@@ -405,9 +406,12 @@ object BookmarkAnnotationsJSON {
     val obj =
       JSONParserUtilities.checkObject(null, node)
 
+    val startOffset = JSONParserUtilities.getIntegerDefault(obj, "startOffset", 0).toLong()
+
     return PlayerPosition(
       chapter = JSONParserUtilities.getIntegerDefault(obj, "chapter", 0),
-      offsetMilliseconds = JSONParserUtilities.getInteger(obj, "time").toLong(),
+      startOffset = startOffset,
+      currentOffset = JSONParserUtilities.getInteger(obj, "time").toLong() - startOffset,
```

Comment:
When deserializing, it looks like `time` is expected to be the offset from the start of the track (which makes sense, to be backward compatible with existing bookmarks). So to get `currentOffset`, the `startOffset` value (if any) is subtracted, which also makes sense. But then, why isn't `startOffset` added on to `currentOffset` when serializing `time`?